R/pars.R

```diff
@@ -155,7 +155,7 @@ spim_pars_pmcmc_save <- function(p, path) {
 ##' @param last_beta_days_ago Number of days in the past for last beta point
 ##'
 ##' @export
-spim_pars_beta <- function(date, last_beta_days_ago = 21) {
+spim_pars_beta <- function(date, last_beta_days_ago = 7) {
```

Comment:
Undo this change (we set `last_beta_days_ago` in the parameters task) and then the failed test should pass